avionics/avionics.ino

```diff
@@ -134,7 +149,7 @@ void setup()
   */
 void loop()
 {
-    static unsigned long timestamp;
+    uint32_t timestamp;
```

Comment:
why change this one at all / why not change all of them?  Does uint32_t make it a static variable?

Reply:
that is a very good question and uint32_t does not make it static
i think the logic is that uint32_t is always going to be 4 bytes, where as unsigned long has just a minimum and technically could change